plugin-server/src/utils/event.ts

```diff
@@ -139,14 +140,45 @@ export function normalizeEvent(event: PluginEvent): PluginEvent {
     return event
 }
 
+export function enrichUserAgent(event: PluginEvent): PluginEvent {
+    const ua = event.properties?.['$user_agent'] || event.properties?.['$raw_user_agent']
+
+    if (event.properties && ua) {
+        const parser = new UAParser(ua)
+        // TODO: Check this is similar to our existing values
+        const device = parser.getDevice().model
```

Comment:
it's not similar :) it's really different.

Reply:
I checked through this in the posthog-js PR

It outputs different values - particularly for OS where it will output gentoo and we output linux

So, we can move to this in future probably to save maintaining the code ourselves but moving in one go would make testing the transition really tricky

Reply:
(btw - this is more of a "hey look how easy this is" than a side quest I'm likely to take on...

Reply:
yep

good to know where it should fit :)